content/reference/endpoints/reviews.md

```diff
@@ -3,44 +3,134 @@
-name             | String  | Read       | Reviewer's name, cropped to show only the initial of the last name.
-rating           | Integer | Read       | Review's rating.
-title            | String  | Read       | Review's title.
------------------|---------|------------|------------
-created_at       | [Time](/reference/enums#formats) | Read       | Review's create time.
-updated_at       | [Time](/reference/enums#formats) | Read       | Review's update time.
-{: class="table table-bordered"}
-  </div>
-  <div class="tab-pane" id="reviews_write" markdown="1">
-Name             | Type    | Read/Write | Description
------------------|---------|------------|------------
-id               | Integer | Read       | Review's id.
------------------|---------|------------|------------
-comment          | Text    | Read/Write | **Required**. Review's comment. (10 min, 2000 max characters)
-country_code     | String  | Read       | Reviewer's country code. e.g: FR
-name             | String  | Read       | Reviewer's name, cropped to show only the initial of the last name.
-rating           | Integer | Read/Write | **Required**. Review's rating. (between 1 and 6)
-title            | String  | Read/Write | Review's title. (3 min, 100 max characters)
------------------|---------|------------|------------
-created_at       | [Time](/reference/enums#formats) | Read       | Review's create time.
-updated_at       | [Time](/reference/enums#formats) | Read       | Review's update time.
-{: class="table table-bordered"}
-  </div>
-</div>
+   <div class="tab-pane active" id="public" markdown="1">
+ Group    | Subgroup            | Name                  | Type    | Read/Write | Description
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | account               | Integer | Read       | Account's ID related to the Review
+          |                     | rental                | Integer | Read       | Rental's ID related to the Review
+          |                     | source                | Integer | Read       | Source's ID related to the Review
+          |                     | reply                 | Integer | Read       | Reply's (ReviewReply)'s ID related to the Review
+          |                     | booking               | Integer | Read       | Booking's ID related to the Review
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | id                    | Integer | Read       | Review's ID. 
+          |                     | title                 | Text    | Read | Title/Summary of the review. 
+          |                     | comment               | Text    | Read | Comment from traveler (guest). 
+          |                     | private_comment       | Text    | Read | Private comment from the traveler (guest) (not meant to be exposed publicly). Requires `reviews_read` or `reviews_write` scope 
```

Comment:
>Requires `reviews_read` or `reviews_write` scope

do we need this comment? It usually is just reflected by displaying or not displaying an attribute in the selected oauth scope table

Reply:
good point